src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,105 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int SIZE_MULTIPLIER = 2;
+    private int threshold;
+    private int size;
+    private Node<K,V>[] table;
+
+    public MyHashMap() {
+        createNewTable();
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size >= threshold) {
```

Comment:
you can delete it and in the resize method the line "size = 0;" replace with check if(size == 0);

Reply:
Can you explain what exactly "if(size == 0)" give me in resize method? According to my implementation every time array need to be resized, size has to be 0. That's because I will transfer all element to new array and size will be increased after every transfer.

Reply:
Sorry, yes you are right. I meant that you can really move if (size >= threshold) to the resize method

Reply:
Alright, i get it now.